pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx

```diff
@@ -71,16 +72,14 @@ export const KubernetesInventory: FC = () => {
             setAddModalVisible(false);
           }}
           render={({
-            form, handleSubmit, valid, pristine
+            handleSubmit, valid, pristine
```

Comment:
We definitely welcome an interface here in order to have types variables. Pls take a look, there is a good [example](https://confluence.percona.com/display/PMM/Working+with+Forms)